docs/Users-Guide.md

```diff
@@ -39,22 +102,24 @@ To play with timesketch without any installation visit [demo.timesketch.org](htt
 tsctl is a command line tool to control timesketch.
 
 Parameters:
-```
+
+```shell
 --config / -c (optional)
 ```
 
 Example
-```
+
+```shell
 tsctl runserver -c /etc/timesketch/timesketch.conf
 ```
 
-
 ### Start timesketch
 
 Will start the timesketch server
 
 Command:
-```
+
+```shell
 tsctl runserver
```

Comment:
are we really running the server through this method? We are more relying on docker containers to do this now

Reply:
true